packages-exp/auth-compat-exp/package.json

```diff
@@ -5,9 +5,8 @@
   "description": "FirebaseAuth compatibility package that uses API style compatible with Firebase@7 and prior versions",
   "author": "Firebase <firebase-support@google.com> (https://firebase.google.com/)",
   "main": "dist/index.node.cjs.js",
-  "browser": "dist/index.esm.js",
-  "module": "dist/index.esm.js",
-  "esm2017": "dist/index.esm2017.js",
+  "browser": "dist/index.esm2017.js",
+  "module": "dist/index.esm2017.js",
```

Comment:
Wonder if we don't want to change compat packages, but I guess it's fine since it's going to be a major release